src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeValueType.php

```diff
@@ -72,7 +73,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                     return;
                 }
 
-                $this->addValueField($event->getForm(), $attribute);
+                $this->addValueField($event->getForm(), $attribute, $localeCode);
```

Comment:
In case when the attribute is not translatable, the `localeCode` will be an empty string. Have you checked if that case doesn't break anything?
Also, I'm guessing we might be missing a scenario for it.